src/pydata_sphinx_theme/toctree.py

```diff
@@ -137,18 +137,22 @@ def generate_header_nav_html(n_links_before_dropdown: int = 5) -> str:
         out = "\n".join(links_solo)
 
         # Wrap the final few header items in a "more" dropdown
-        links_dropdown = links_html[n_links_before_dropdown:]
+        links_dropdown = [
+            html.replace("nav-link", "nav-link dropdown-item")
+            for html in links_html[n_links_before_dropdown:]
+        ]
```

Comment:
I think I want to re-think the way I'm adding this CSS class to the markup because this code seems a little brittle to me. I mean, it **seems** like we can count on each 'a' element having the "nav-link" class (and nothing else), and if that assumption holds, this code should be safe, but it still has some kind of code smell to me.

Reply:
I detected the same smell, but let's punt on this for now. It works and I agree that it seems like it should be fine.

Reply:
Like this? 

```suggestion
        links_dropdown = [
            # 🐲 brittle code, relies on the assumption that the code above
            # gives each link in the nav a `nav-link` CSS class            html.replace("nav-link", "nav-link dropdown-item")
            for html in links_html[n_links_before_dropdown:]
        ]
```

Reply:
yes except it looks like you inadvertantly commented out some real code in the process?

Reply:
Grrrr, trying again. 

```suggestion
        links_dropdown = [
            # 🐲 brittle code, relies on the assumption that the code above
            # gives each link in the nav a `nav-link` CSS class        
            html.replace("nav-link", "nav-link dropdown-item")
            for html in links_html[n_links_before_dropdown:]
        ]
```